docs/cli/plugin_implementation_guide.md

```diff
@@ -49,6 +49,17 @@ To add the admin repository use `tanzu plugin repo add -n admin -b tanzu-cli-adm
 
 To add the builder plugin use `tanzu plugin install builder`
 
+#### Installing Admin Plugins
+
+With the [context-aware plugin discovery](/docs/design/context-aware-plugin-discovery-design.md) been enabled within Tanzu CLI admin plugins can be installed from local source as mentioned below:
+
+1. Download the admin plugin tarball or zip file from the release artifacts and extract it
+1. Run `tanzu plugin lsit --local /path/to/admin-plugins/linux-amd64-admin` to list available admin plugins
+1. Run `tanzu plugin install all --local /path/to/admin-plugins/linux-amd64-admin` command to install all admin plugins. User can user `plugin-name` instead of `all` to install specific plugin
+1. Run `tanzu plugin list` to verify the installed plugins
+
+NOTE: We are working on enhacing this user experience by publishing admin artifacts as discovery source as part of OCI image. This will improve the experience of users. More details is in [this issue](https://github.com/vmware-tanzu/tanzu-framework/issues/1376).
+
 ### Plugin Discovery Source
 
 Discovery is the interface to fetch the list of available plugins, their supported versions and how to download them either standalone or scoped to a context(server). E.g., the CLIPlugin API in a management cluster, OCI based plugin discovery for standalone plugins, a similar REST API etc. Having a separate interface for discovery helps to decouple discovery (which is usually tied to a server or user identity) from distribution (which can be shared).
```

Comment:
I don't quite follow the "E.g.,...." sentence

Reply:
added more details to the sentence